keras/random/seed_generator.py

```diff
@@ -89,6 +89,9 @@ def next(self, ordered=True):
             self.state.assign((seed_state + 1) * 5387 % 933199)
         return new_seed_value
 
+    def get_config(self):
+        return {"_initial_seed": self._initial_seed, "state": self.state}
```

Comment:
This should only contain "seed" (constructor arg)

Reply:
Thanks Francois! I updated this and I also added `backend` that could be passed in the `kwargs`.